src/land/owp_snow_obs.py

```diff
@@ -206,9 +203,8 @@ def _read(self):
             self.data[self.var_dict[ioda_var]['qcKey']] = (
                 mask_nans(obs_df[f'PreQC {obs_var}'].values * conv_fact))
 
-        nlocs = len(self.data[('datetime', 'MetaData')])
-        dim_dict['nlocs'] = nlocs
-        attr_data['nlocs'] = np.int32(nlocs)
+        Location = len(self.data[('datetime', 'MetaData')])
```

Comment:
`dateTime` again

Reply:
Also can you condense these two lines into one?
`dim_dict['Location'] = len(self.data[('dateTime', 'MetaData')])`

Reply:
Yes, corrected and condensed.